code/edgebridge/build.gradle.kts

```diff
@@ -0,0 +1,39 @@
+import com.adobe.marketing.mobile.gradle.BuildConstants
+
+plugins {
+    id("aep-library")
+}
+
+val mavenCoreVersion: String by project
+val androidxAnnotationVersion: String by project
+
+aepLibrary {
+    namespace = "com.adobe.marketing.mobile.edge.bridge"
+
+    publishing {
+        gitRepoName = "aepsdk-edgebridge-android"
+        addCoreDependency(mavenCoreVersion)
+    }
+
+    enableSpotless = true
+    enableSpotlessPrettierForJava = true
+}
+
+dependencies {
+    // TODO: Use 3.x versions for testing
+    implementation("com.adobe.marketing.mobile:core:$mavenCoreVersion")
+    implementation("androidx.annotation:annotation:$androidxAnnotationVersion")
+
+    testImplementation(BuildConstants.Dependencies.ANDROIDX_TEST_EXT_JUNIT)
```

Comment:
Most of these standard dependencies are [included by the plugin](https://github.com/adobe/aepsdk-commons/blob/dev/android/aepsdk-gradle-plugin/aepsdk-gradle-plugin/src/main/kotlin/com/adobe/marketing/mobile/gradle/AEPLibraryPlugin.kt#L250) to ensure consistency across all our projects. Only include the ones that are unique to this repository.